requirements-test.txt

```diff
@@ -314,7 +314,7 @@ ruamel.yaml.clib==0.2.0 \
 ruamel.yaml==0.16.10 \
```

Comment:
Same as the comment about the recommendation in their docs

Reply:
Same as above.